tls/s2n_connection.h

```diff
@@ -365,6 +320,40 @@ struct s2n_connection {
     /* Cookie extension data */
     struct s2n_stuffer cookie_stuffer;
 
+    /* Has the user set their own I/O callbacks or is this connection using the
+     * default socket-based I/O set by s2n */
+    unsigned managed_send_io:1;
+    unsigned managed_recv_io:1;
```

Comment:
The bitfields were initially split into two different fields in this structure because of errors thrown by the SAW proofs. With bitfields now properly represented, can you consolidate the fields? You can leave them where they are (at the beginning of the structure) and just remove the comments about order mattering.

Reply:
Ah, I had assumed there was a reason behind their original order. But I'm just as happy to leave them all at the front of the structure as they currently are. I've pushed a change which does this, removing the obsolete comments about their order in the process.